c_src/stb_image_nif.c

```diff
@@ -0,0 +1,264 @@
+#include <erl_nif.h>
+#define STB_IMAGE_IMPLEMENTATION
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include <stb_image.h>
+#include <stb_image_write.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#define MAX_NAME_LENGTH 9999
```

Comment:
```suggestion
#define MAX_NAME_LENGTH 2048
```

Reply:
I think `MAX_TYPE_LENGTH` should be `4`? because [`enif_get_string`](https://www.erlang.org/doc/man/erl_nif.html#enif_get_string) also count the `NULL` at the end of the string.

Reply:
Good find. I will fix it locally!